spec/lib/task_helpers/exports/policies_spec.rb

```diff
@@ -0,0 +1,91 @@
+describe TaskHelpers::Exports::Policies do
+  let(:guid) { "a61314d5-67bd-435f-9c82-b82226e0a7fe" }
+  let(:guid2) { "ac7e2972-f2b2-4ebe-b29d-97eefaac7615" }
+
+  let(:profile_export_attrs) do
+    [
+      {
+        "MiqPolicy" => {
+          "name"             => "a61314d5-67bd-435f-9c82-b82226e0a7fe",
+          "description"      => "Test Compliance Policy",
+          "expression"       => nil,
+          "towhat"           => "Vm",
+          "guid"             => "a61314d5-67bd-435f-9c82-b82226e0a7fe",
+          "created_by"       => nil,
+          "updated_by"       => nil,
+          "notes"            => nil,
+          "active"           => true,
+          "mode"             => "compliance",
+          "read_only"        => nil,
+          "MiqPolicyContent" => [],
+          "Condition"        => []
+        }
+      }
+    ]
+  end
+
+  let(:policy_create_attrs) do
+    {
+      :description => "Test Compliance Policy",
+      :name        => guid,
+      :guid        => guid,
+      :mode        => "compliance",
+    }
+  end
+
+  let(:profile2_export_attrs) do
+    [
+      {
+        "MiqPolicy" => {
+          "name"             => "ac7e2972-f2b2-4ebe-b29d-97eefaac7615",
+          "description"      => "Test Compliance Policy 2",
+          "expression"       => nil,
+          "towhat"           => "Host",
+          "guid"             => "ac7e2972-f2b2-4ebe-b29d-97eefaac7615",
+          "created_by"       => nil,
+          "updated_by"       => nil,
+          "notes"            => nil,
+          "active"           => true,
+          "mode"             => "control",
+          "read_only"        => true,
+          "MiqPolicyContent" => [],
+          "Condition"        => []
+        }
+      }
+    ]
+  end
+
+  let(:policy2_create_attrs) do
+    {
+      :description => "Test Compliance Policy 2",
+      :name        => guid2,
+      :guid        => guid2,
+      :towhat      => "Host"
+    }
+  end
+
+  before(:each) do
+    FactoryGirl.create(:miq_policy, policy_create_attrs)
+    FactoryGirl.create(:miq_policy_read_only, policy2_create_attrs)
+    @export_dir = Dir.mktmpdir('miq_exp_dir')
+  end
+
+  after(:each) do
+    FileUtils.remove_entry(@export_dir)
+  end
+
+  it 'should export user policies to a given directory' do
```

Comment:
Minor change, but I believe we've stopped prefixing the descriptions with "should", so this would be better written as something like:
`it 'exports user policies to a given directory'`